documentation/docs/specs/did/iota_did_method_spec.md

```diff
@@ -295,44 +295,7 @@ In order to create a valid Integration DID message, the following steps are to b
 
 ### Delete
 
-In order to deactivate a DID document, a valid Integration DID message must be published that removes all content from a DID Document, effectively deactivating the DID Document. Keep in mind that this operation is irreversible.
-
-## IOTA Identity standards
```

Comment:
I'd rather keep this around and be updated to what we have now, which is not a lot. But a section on "Verification Methods Types" and "Standardized Services" will be a good reminder to expand on those in the future. Keeping them as part of the DID Method should encourage more interoperability with alternative implementations of this method.